src/mapInteractor.js

```diff
@@ -923,6 +936,12 @@ var mapInteractor = function (args) {
     if (m_paused) {
       return;
     }
+    /* In some scenarios, we get both a tap event and then, somewhat later, a
+     * set of mousedown/mouseup events.  Ignore the spurious down/up set if we
+     * just handled a tap. */
+    if (m_touchHandler && m_touchHandler.lastEventType === 'tap' && (new Date()).valueOf() - m_touchHandler.lastTime < 1000) {
```

Comment:
worth wrapping this long line? May be we could have a test for long lines - if makes sense we could allow devs to have length of 79/80 or 120 or not :)